script/millerQuestions.sh

```diff
@@ -4,6 +4,7 @@ set -x
 set -e
 set -u
 set -o pipefail
+alias mlr='mlr.linux.x86_64'
```

Comment:
@NikosAlexandris my mlr executable is `mlr` not `mlr.linux.x86_64`. Using this line will I have problems?

Reply:
Oops! This is something local! I shouldn't have committed it.